src/Pages/Auth/Login.jsx

```diff
@@ -1,76 +1,95 @@
-import React from "react"
-import Navbar from "../../Component/Navbar/Navbar";
-import { Link } from "react-router-dom";
-import { useState } from "react";
-import { useAuth } from "../../Context/Auth/Auth";
+import React from 'react';
+import Navbar from '../../Component/Navbar/Navbar';
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { useAuth } from '../../Context/Auth/Auth';
 import { useNavigate } from 'react-router-dom';
 function Login() {
+  const { fetchLoginDetails } = useAuth();
+  const [newClient, setNewClient] = useState({ email: '', password: '' });
 
+  const navigate = useNavigate();
 
-const {fetchLoginDetails} = useAuth();
-const [newClient,setNewClient] = useState({email:"",password:""})
-
-const navigate = useNavigate();
-
-return (
-<>
-    <main>
-
+  return (
+    <>
+      <main>
         <Navbar></Navbar>
-        <section className=" box-shadow-1 m-auto m-y-11  wt-30 flex flex-column items-center pd-8 ">
-            <div className="text-xm text-semi">
-                <span>Login</span>
+        <section className=' box-shadow-1 m-auto m-y-11  wt-30 flex flex-column items-center pd-8 '>
+          <div className='text-xm text-semi'>
+            <span>Login</span>
+          </div>
+          <form
+            onSubmit={(e) => {
+              e.preventDefault();
+              fetchLoginDetails(newClient.email, newClient.password);
+              navigate('/');
+            }}
+          >
+            <div className=' bg-black-0 flex flex-column  m-t-8 wt-100 '>
+              <label for='email' className='bg-white-0 '>
```

Comment:
suggestion: you can make an input component and reuse it.

Reply:
okk .
Component suggestion.